sympy/physics/units/tests/test_prefixes.py

```diff
@@ -17,15 +17,16 @@ def test_prefix_operations():
 
     dodeca = Prefix('dodeca', 'dd', 1, base=12)
 
-    assert m * k == 1
+    assert m * k == S.One
```

Comment:
since 1==S.One, this would need to be tested using `is` instead `==`.

Reply:
I changed accordingly. Thank you.